src/views/CTLViewsHoC.jsx

```diff
@@ -203,6 +203,7 @@ const CTLViewsHoC = (WrappedView) => {
 
 		return (
 			<div>
```

Comment:
pls change this div to a fragment `<>`

Reply:
I'm not sure why, but when I change the divs to a fragment, it breaks the unit tests that handle finding the text element: "This would be the disease component." in the page.